keps/sig-scheduling/3280-guarantee-pdb-when-preemption-happens/kep.yaml

```diff
@@ -0,0 +1,44 @@
+title: Guarantee PodDisruptionBudget When Preemption Happens
+kep-number: 3280
+authors:
+  - "@denkensk"
+owning-sig: sig-scheduling
+participating-sigs:
+  - sig-apps
+  - sig-auth
+status: implementable
+creation-date: 2023-01-17
+reviewers:
+  - "@ahg-g"
+  - "@alculquicondor"
+  - "@liggitt"
```

Comment:
any participating SIG should have an approve, so you can move these ones

You still need one from SIG Apps too

Reply:
Because I do not need to update PDBs api, does it still need to be approved by sig-app?

Reply:
Ah, that's a good point. Then please remove sig-apps from here.

@liggitt should we be adding sig-scheduling and sig-auth tags to this folder? https://github.com/kubernetes/kubernetes/tree/master/staging/src/k8s.io/api/scheduling

Reply:
> Then please remove sig-apps from here.

Done